packages/rempl-apollo-devtools/src/publisher/publishers/apollo-cache-publisher.ts

```diff
@@ -61,18 +44,17 @@ export class ApolloCachePublisher {
       return;
     }
 
-    if (this.activeClient?.clientId !== activeClient.clientId) {
-      this.lastCacheHistory = {};
-    }
-    this.activeClient = activeClient;
-
     const serializedCacheObject = this.serializeCacheObject(activeClient);
 
     if (!serializedCacheObject) {
       return;
     }
 
-    if (!this.hasCacheChanged(this.lastCacheHistory, serializedCacheObject)) {
+    if (this.activeClient?.clientId !== activeClient.clientId) {
```

Comment:
Can we compare instances? I believe clientId is tmp-specfic

```suggestion
    if (this.activeClient !== activeClient) {
```